loopdb/sqlc/queries/batch.sql

```diff
@@ -73,11 +73,7 @@ FROM
 JOIN
         sweeps ON sweep_batches.id = sweeps.batch_id
 WHERE
-        sweeps.swap_hash = $1
-AND
-        sweeps.completed = TRUE
-AND
-        sweep_batches.confirmed = TRUE;
+        sweeps.swap_hash = $1;
```

Comment:
This looks more simple, not every parent has to be a completed one

Reply:
The reason of that extra condition was that it was assumed (wrongly) that the parent is confirmed by the time the sweep is confirmed. Given that we move the assumption check before we monitor it is not needed in the DB.